_posts/2025-05-04-update.md

```diff
@@ -7,6 +7,11 @@ slug: 2025-05-04-update
 
 ## Developer News
 
+Joel Speed is being nominated as a technical lead for SIG Cloud Provider. This was discussed in the [April 23, 2025 SIG Cloud Meeting](https://docs.google.com/document/d/1OZE-ub-v6B8y-GuaWejL-vU_f9jsjBbrim4LtTfxssw/edit?tab=t.0#heading=h.y29cpq3g6i2i). Joel has been active in SIG Cloud Provider for about four years, bringing detailed and helpful insights. There is no other change to the SIG Cloud Provider leadership. Michael McCune will stay as a co-chair and technical lead, Bridget Kromhout will stay as co-chair, and Walter Fender will stay as a technical lead.
```

Comment:
For future reference, I think such updates could be shorter. See the SIG Scheduling updates in the [April 13th issue](https://lwkd.info/2025/20250416) for reference. 

CC: @jberkus

Reply:
I'm good with merging this, this comment was for future SIG leadership updates.